source/frontend/src/features/mapSideBar/compensation/list/CompensationListContainer.tsx

```diff
@@ -222,17 +263,19 @@ export const CompensationListContainer: React.FC<ICompensationListContainerProps
   };
 
   React.useEffect(() => {
-    if (compensations === undefined || sidebar.staleFile) {
+    if (fileCompensationResults === null || sidebar.staleFile) {
```

Comment:
please use the exists() function to avoid cases where this is unexpectedly undefined.